cpg_utils/hail_batch.py

```diff
@@ -337,16 +350,19 @@ def image_path(suffix: str) -> str:
     Parameters
     ----------
     suffix : str
-        Describes the location within the registry.
+        Describes the location within the registry, or key in the `images` config
```

Comment:
Should we always require that images are specified in the config, i.e. only support keys here?

Reply:
Sure. Will have to send another PR in `automated-interpretation-pipeline` that calls image_path and reference_path

Reply:
Hmm, in this case `automated-interpretation-pipeline` should somehow know `config-template.toml` from cpg-pipes

Reply:
Should we move `config-template.toml` here?

Reply:
https://github.com/populationgenomics/automated-interpretation-pipeline/pull/43

Reply:
Yes, I think moving it here is fine -- we wanted to merge `cpg-pipes` and `cpg-utils` anyway, right?